compiler/src/dmd/cli.d

```diff
@@ -465,7 +465,57 @@ dmd -cov -unittest myprog.d
             "print help and exit"
         ),
         Option("I=<directory>",
-            "look for imports also in directory"
+            "look for imports also in directory",
+            q"{$(P Adds $(I directory) to the list of paths to be searched for imports.
+             Multiple `-I`'s can be used, and the paths are searched in the same order.
+
+             The process of finding a module during the resolution of an import
```

Comment:
Doesn't look like this adds a line break in the generated web page.  Will look to see what the macro is to add one.